packages/protocol/test/stability/stabletoken.ts

```diff
@@ -313,12 +313,9 @@ contract('StableToken', (accounts: string[]) => {
     })
 
     describe('#when there is 0.5% weekly inflation', () => {
-      beforeEach(async () => {
+      it('should return depreciated balance value', async () => {
```

Comment:
Personally I don't mind having a `beforeEach` here even though there's just one `it`. It's a nice semantic separation between test setup and the actual test. And makes it easier to add more tests in the future.